tests/test_new_command.rs

```diff
@@ -106,6 +106,154 @@ fn test_new_merge() {
     "###);
 }
 
+// All tests test_new_insert_* start with the same initial graph, which is
+// checked once in `test_new_insert_setup()` below.
+#[test]
+fn test_new_insert_setup() {
+    // Check that the initial setup starts in the expected state.
+    let test_env = TestEnvironment::default();
+    test_env.jj_cmd_success(test_env.env_root(), &["init", "repo", "--git"]);
+    let repo_path = test_env.env_root().join("repo");
+    setup_before_insertion(&test_env, &repo_path);
```

Comment:
I think it's better to check the initial state in each test for readability instead of having this test.

Reply:
That's what I had first but decided to cut it. Restored, it is indeed more readable.

Reply:
Is there a point to still have `test_new_insert_setup`?

Reply:
No, but it has been removed already.

Reply:
I see, it's removed in the next commit.

Reply:
Good catch, thanks, it should have been removed from the first, fixing it.